### PR TITLE
Issue 270: Fix conda package dependencies (jax version)

### PR DESCRIPTION
**Pull request by willis-richard**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

PR to address https://github.com/deepmind/chex/issues/270

### CODE REVIEW

**Comment by willis-richard** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,9 +23,9 @@ requirements:
     - python >=3.6
     - absl-py >=0.9.0
     - dm-tree >=0.1.5
-    - jax >=0.1.55
+    - jax >=0.4.6
     - jaxlib >=0.1.37
-    - numpy >=1.18.0
+    - numpy >=1.25.0
```
Comment (PhilipVinc) [inline recipe/meta.yaml]: you sure about that one? it's super recent. Maybe it risk colliding with other package dependencies.
I think recent jax versions only require numpy `1.22` or higher, so you could set this?

```suggestion
    - numpy >=1.22.0
```

**Review by PhilipVinc — COMMENTED** [review]
apart from this minor thing, it's good.

**Review by PhilipVinc — APPROVED** [review]

**Comment by PhilipVinc** [comment]
@conda-forge-admin, please rerender

**Comment by PhilipVinc** [comment]
thanks!

File: recipe/meta.yaml (hunk 1 above)
Comment (willis-richard) [reply recipe/meta.yaml]: Just FYI, I am not the right person to know about numpy versions, but ]I took that number from https://github.com/deepmind/chex/blob/master/requirements/requirements.txt.

**Comment by willis-richard** [comment]
Hmm, I think there is a bug. On conda forge, the version has jumped 0.1.8 ->0.1.81 instead of 0.1.8.1.
https://anaconda.org/conda-forge/chex/files

Also, I replied about why I chose numpy>1.25.

**Comment by willis-richard** [comment]
Oh, no wait, that increase to 0.1.81 happened before this PR.

**Comment by PhilipVinc** [comment]
It is really the versino. I think the mistake was upstream.

As for the dependency, I don't think it's terribly important but you can open another PR if you wish

**Pull request merged**